openfisca_uk/variables/finance/benefit/family/universal_credit.py

```diff
@@ -506,7 +506,18 @@ def formula(benunit, period, parameters):
             "self_employment_income",
             "miscellaneous_income",
         ]
-        earned_income = aggr(benunit, period, INCOME_COMPONENTS)
+        personal_earned_income = add(
+            benunit.members, period, INCOME_COMPONENTS
+        )
+        personal_mif_capped_income = where(
+            benunit.members("UC_MIF_applies", period),
+            max_(
```

Comment:
Maybe make this a separate variable (Python, not OpenFisca)

Reply:
Done